test/libsolidity/syntaxTests/specialFunctions/types_with_unspecified_encoding_internal_functions.sol

```diff
@@ -1,12 +1,10 @@
 contract C {
     function f() public pure {
-        bytes32 h = keccak256(abi.encodePacked(keccak256, f, this.f.gas, blockhash));
+        bytes32 h = keccak256(abi.encodePacked(keccak256, f, blockhash));
```

Comment:
Did you/Can you create an issue for investigating ``this.f.gas(2)`` and ``this.f{gas: 2}`` on develop?

Reply:
Yes. Also I need to update some .sol files inside test/compilationTests.

Reply:
New issue based on the file https://github.com/ethereum/solidity/issues/8590